api-explorer/README.md

```diff
@@ -0,0 +1,10 @@
+# Hosted API Explorer
+
+This directory contains [swagger-ui](https://github.com/swagger-api/swagger-ui) assets that serve as a hosted API Explorer for LoopBack 4 applications. The UI will be available at https://loopback.io/api-explorer?url=<swagger-spec-url>.
+
+To upgrade swagger-ui to a newer version:
+
+1. Update the [`swagger-ui-dist`](https://www.npmjs.com/package/swagger-ui-dist) version in [package.json](https://github.com/strongloop/loopback.io/blob/gh-pages/package.json). You can use `npm update swagger-ui-dist`.
+
+2. Run `npm install` which in turn executes `./update-swagger-ui.sh` to copy files into `api-explorer`.
+
```

Comment:
IIUC, the third step is to add any new or modified files to git and send a PR with the changes. Can you mention this in the instructions please?

Ideally, we should be deleting files that are no longer part of swagger-ui too - I think that's something to handle in `update-swagger-ui` script, for example by removing everything from the `api-explorer` directory except "our" files (README.md, index.loopback.html).